wee_slack.py

```diff
@@ -4000,22 +4001,38 @@ def command_thread(data, current_buffer, args):
 
 command_thread.completion = '%(threads)'
 
+
 @slack_buffer_required
 @utf8_decode
 def command_reply(data, current_buffer, args):
     """
-    /reply <count/message_id> <text>
-    Reply in a thread on the message. Specify either the message id
-    or a count upwards to the message from the last message.
+    /reply [-alsochannel] <count/message_id> <text>
```

Comment:
Since the count/id is optional now, this should be changed to ` [count/message_id]`. Same with the usage-line below.

Reply:
How's this?  I think it makes it more explicit which form is allowed when.

Reply:
Yeah, that would be good, except that it looks a bit weird in the help text inside weechat, because the code which generates the help text from the docstring expects the command first.

With this `/help reply` outputs this:

```
23:16 [python/slack]  /reply  in a channel buffer:
23:16 
23:16 /reply [-alsochannel] <count/message_id> <text>
23:16 Reply in a thread on the message. Specify either the message id or a count
23:16 upwards to the message from the last message.
23:16 
23:16 When in a thread buffer:
23:16 /reply [-alsochannel] <text>
23:16 Reply to the current thread.  This can be used to send the reply to the
23:16 rest of the channel.
```

One possible fix for it would be keeping `/reply [-alsochannel] [count/message_id] <text>` as the first line, and then include what you wrote below.